core/src/main/scala/kafka/cluster/Broker.scala

```diff
@@ -93,41 +106,53 @@ object Broker {
    *               id (int), number of endpoints (int), serialized endpoints
    * @return broker object
    */
-  def readFrom(buffer: ByteBuffer): Broker = {
+  def readFrom(buffer: ByteBuffer, expectRack: Boolean = true): Broker = {
```

Comment:
Yes and the KIP should be updated to remove the point about updating `Broker.writeTo`.